build/templates/run-service-bus-test-worker.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  buildVersion: ''
+  healthPort: ''
+
+steps:
+- bash: |
+    if [ -z "$BUILD_VERSION" ]; then
+      echo "##vso[task.logissue type=error;]Missing template parameter \"manualTriggerVersion\""
+      echo "##vso[task.complete result=Failed;]"
+    fi
+    if [ -z "$HEALTH_PORT" ]; then
+      echo "##vso[task.logissue type=error;]Missing template parameter \"healthPort\""
+      echo "##vso[task.complete result=Failed;]"
+    fi
+  env:
+    BUILD_VERSION: ${{ parameters.buildVersion }}
+    HEALTH_PORT: ${{ parameters.healthPort }}
+  displayName: Check for required parameters in YAML template
+- task: Docker@1
+  displayName: 'Build Docker image for Service Bus test worker'
+  inputs:
+    dockerFile: src/Arcus.Messaging.Tests.Worker/Dockerfile
+    imageName: 'Arcus.Messaging.Tests.Worker:${{ parameters.buildVersion }}'
+    useDefaultContext: false
+    buildContext: src
+- task: Docker@1
+  displayName: 'Run Service Bus test worker from Docker image'
+  inputs:
+    command: 'Run an image'
+    imageName: 'Arcus.Messaging.Tests.Worker:${{ parameters.buildVersion }}'
+    containerName: 'Arcus.Messaging.Tests.Worker'
```

Comment:
Maybe also create a `containerName` parameter for this and assign `Arcus.Messaging.Tests.Worker` as its default value?

Reply:
Updated!